definitions/subworkflows/cnvkit_single_sample.cwl

```diff
@@ -73,16 +74,6 @@ steps:
             cns_file: cnvkit_main/tumor_segmented_ratios
             male_reference: male_reference
             cnr_file: cnvkit_main/tumor_bin_level_ratios
-            output_name: 
-                source: cnvkit_vcf_name
-                valueFrom: |
-                    ${  
-                        if(inputs.output_name) {
-                            return inputs.output_name;
-                        }   
-                        else {
-                            return inputs.tumor_bam.nameroot + ".cnvkit.vcf"
-                        }   
-                    }
```

Comment:
Are other pipelines relying on a specified output file name? If not, this seems fine, but I'm curious why the prior behavior, ie. specfied output names.

Reply:
I think they were at the time this was written, but I don't see any place where it matters now, so it should be fine.